### DISCUSSION TITLE
Atualização v6.4

### DISCUSSION THREAD

**Discussion by jgsat** [sanni/cartreader#195] [General Chat]
Não estou conseguindo atualizar para v6.4,não inicia.

**Comment by sanni** [comment]
Please follow every step of this guide and make a screenshot of your error: https://github.com/sanni/cartreader/wiki/How-to-flash-the-Arduino

**Comment by jgsat** [comment]
Aparece essa mensagem nao consigo carregar o scket.

Em qua, 14 de jul de 2021 às 07:25, sanni ***@***.***>
escreveu:

**Comment by sanni** [comment]
Try formatting the SD card to exFAT.

**Comment by jgsat** [comment]
formatei nao aconteceu nada

Em qua., 14 de jul. de 2021 às 19:55, sanni ***@***.***>
escreveu:

**Comment by sanni** [comment]
Did you copy these files to the SD card? https://github.com/sanni/cartreader/tree/master/sd

**Comment by jgsat** [comment]
Já formatei o cartão  em exfat e copiei os arquivo na pasta raiz do cartão
mais acontece a mesmo erro .Mais na versão 5.5 carregar o sketch normal .

Em qui, 15 de jul de 2021 às 06:03, sanni ***@***.***>
escreveu:
